maps/site53/structures/closets/medical.dm

```diff
@@ -1,5 +1,5 @@
 /*
- * Torch Medical
+ * Torch -/- Medical
```

Comment:
Delete this entire comment.

Reply:
okay- the codebase is full of old things, i just wanted to preserve it's oldness like elite said.